lib/node_modules/@stdlib/math/base/special/hyp2f1/README.md

```diff
@@ -0,0 +1,162 @@
+```
+
+<!-- <div class="equation" align="center" data-raw-text="{}_2F_1(a, b; c; x) = \sum_{n=0}^{\infty} \frac{(a)_n (b)_n}{(c)_n} \frac{x^n}{n!} = 1 + \frac{a b}{c} x + \frac{a(a+1) b(b+1)}{c(c+1)} \frac{x^2}{2!} + \frac{a(a+1)(a+2) b(b+1)(b+2)}{c(c+1)(c+2)} \frac{x^3}{3!} + \cdots" data-equation="eq:hypergeometric_function">
+    <img src="https://cdn.jsdelivr.net/gh/stdlib-js/stdlib@bb29798906e119fcb2af99e94b60407a270c9b32/lib/node_modules/@stdlib/math/base/special/hyp2f1/docs/img/equation_hypergeometric_function.svg" alt="Hypergeometric function.">
+    <br>
+</div> -->
+
+<!-- </equation> -->
+
+and is undefined (or infinite) if `c` equals a non-positive integer.
+
+Here `(q)ₙ` is the (rising) [Pochhammer symbol][pochhammer-symbol], which is defined by:
+
+<!-- <equation class="equation" label="eq:pochhammer_symbol" align="center" raw="(q)_n = \begin{cases} 1 & n = 0 \\ q(q+1) \cdots (q+n-1) & n > 0 \end{cases}" alt="Pochhammer symbol."> -->
+
+```math
+(q)_n = \begin{cases} 1 & n = 0 \\ q(q+1) \cdots (q+n-1) & n > 0 \end{cases}
+```
+
+<!-- <div class="equation" align="center" data-raw-text="(q)_n = \begin{cases} 1 & n = 0 \\ q(q+1) \cdots (q+n-1) & n > 0 \end{cases}" data-equation="eq:pochhammer_symbol">
+    <img src="https://cdn.jsdelivr.net/gh/stdlib-js/stdlib@bb29798906e119fcb2af99e94b60407a270c9b32/lib/node_modules/@stdlib/math/base/special/hyp2f1/docs/img/equation_pochhammer_symbol.svg" alt="Pochhammer symbol.">
+    <br>
+</div> -->
+
+<!-- </equation> -->
+
+For `|x| >= 1`, the function can be [analytically continued][analytic-continuation] using functional identities and transformation formulas.
+
+</section>
+
+<!-- /.intro -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var hyp2f1 = require( '@stdlib/math/base/special/hyp2f1' );
+```
+
+#### hyp2f1( a, b, c, x )
+
+Evaluates the [Hypergeometric function][hypergeometric-function].
+
+```javascript
+var v = hyp2f1( 1.0, 1.0, 1.0, 0.0 );
+// returns 1.0
+
+v = hyp2f1( 10.0, 7.4, -1.8, -0.99 );
+// returns ~0.423
+
+v = hyp2f1( 3.0, 4.0, 7.0, 1.0 );
+// returns +Infinity
+
+v = hyp2f1( NaN, 3.0, 2.0, 0.5 );
+// returns NaN
+```
+
+</section>
+
+<!-- /.usage -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var linspace = require( '@stdlib/array/base/linspace' );
+var hyp2f1 = require( '@stdlib/math/base/special/hyp2f1' );
+
+var a = linspace( -50.0, 50.0, 100 );
+var b = linspace( -50.0, 50.0, 100 );
+var c = linspace( -50.0, 50.0, 100 );
+var x = linspace( -50.0, 50.0, 100 );
+
+var i;
+for ( i = 0; i < x.length; i++ ) {
+    console.log( 'a: %d, b: %d, c: %d, x: %d, 2F1(a,b;c;x): %d', a[ i ], b[ i ], c[ i ], x[ i ], hyp2f1( a[ i ], b[ i ], c[ i ], x[ i ] ) );
+}
+```
+
+</section>
+
+<!-- /.examples -->
+
+<!-- Section for related `stdlib` packages. Do not manually edit this section, as it is automatically populated. -->
+
+<section class="related">
+
+* * *
+
+## See Also
```

Comment:
For a new package, these will get blown away, as the related package section is automatically generated by a centralized project database. As such, make a mental note, and after this PR is merged, we can add the function to the global namespace.

Reply:
Sorry. I was not explicit. This list and the associated links should be removed.